src/components/multiscreen/multiscreen.scss

```diff
@@ -0,0 +1,50 @@
+@import '~styles/variables';
+
+$multiscreen-player-height: 58px;
+
+.multiscreenWrapper {
+  position: relative;
+
+  .active {
+    background-color: $tone-6-color;
+  }
+  .multiscreenPlayersWrapper {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    bottom: calc(-1 * (10px + #{$multiscreen-player-height}));
+    visibility: hidden;
+
+    &.visible {
+      visibility: visible;
+    }
+
+    .multiscreenPlayer {
+      display: flex;
+      flex-direction: column;
+      position: relative;
+      width: 104px;
+      height: $multiscreen-player-height;
+      border-radius: $roundness-1;
+      overflow: hidden;
+      margin: 0px 4px;
+
+      .multiscreenButtonsWrapper {
+        position: absolute;
+        top: 0;
+        right: 0;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        padding: 0px 16px;
+        justify-content: center;
+        align-items: center;
+        background: linear-gradient(180deg, rgba(0, 0, 0, 0.7) 0%, rgba(0, 0, 0, 0) 100%);
```

Comment:
linear-gradient needs vendor prefixes for other browsers.
you can use autoprefixer to avoid adding them manually (autoprefixer also has a vscode extension).

Reply:
@SivanA-Kaltura 
You can use `linear-gradient`without prefixes and it will be supported by all modern browsers. For more detailed info see: http://caniuse.com/#search=linear-gradient

Reply:
you're right. i wasn't looking at the updated table.